211018_4win_main.py

```diff
@@ -117,17 +110,18 @@ def row_win(player):
 y = 5
 while play:
     if turn % 2 != 0:
-        inputP1 = int(input("_"))
+        inputP1 = int(input("Player 1 "))

```

Comment:
It's more descriptive, but in the end needs to be adjusted again.
Codewars test do not like inputs.

Reply:
yeah, I changed it for easier debugging